src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ReleaseDateService.kt

```diff
@@ -10,16 +10,15 @@ import java.time.LocalDate
 
 @Service
 class ReleaseDateService(
-  private val bankHolidayService: BankHolidayService,
   private val clock: Clock,
+  private val workingDaysService: WorkingDaysService,
   @Value("\${maxNumberOfWorkingDaysAllowedForEarlyRelease:3}") private val maxNumberOfWorkingDaysAllowedForEarlyRelease: Int = 3,
-  @Value("\${maxNumberOfWorkingDaysToTriggerAllocationWarningEmail:6}") private val maxNumberOfWorkingDaysToTriggerAllocationWarningEmail: Int = 6,
+  @Value("\${maxNumberOfWorkingDaysToTriggerAllocationWarningEmail:5}") private val maxNumberOfWorkingDaysToTriggerAllocationWarningEmail: Int = 5,
```

Comment:
Adjusted the logic on this as it should have been 5 days or less as explained in CVSL-1344